trunk/Admin/js/admin-quickbar-admin.js

```diff
@@ -142,13 +145,45 @@ let AdminQuickbar = function () {
       $('body').addClass('admin-quickbar-is-overlap');
     }
 
-    if (localStorage.adminQuickbarDarkmode === 'true') {
-      $('.admin-quickbar-darkmode input').prop('checked', true);
-      $('body').addClass('admin-quickbar-is-darkmode');
-    }
+    self.checkDarkmode();
 
     // init hidden post types
     self.initHiddenPostTypes();
+
+    self.setLanguageSwitchActiveClass();
+    self.hideByLanguage();
+  };
+
+  self.checkDarkmode = function() {
```

Comment:
I think this function is inappropriately named. It doesn't exclusively check for dark mode. It checks every mode.

Reply:
Yes, it´s changed now